addons/pos_self_order/controllers/orders.py

```diff
@@ -36,8 +36,6 @@ def process_order(self, order, access_token, table_identifier, device_type):
         line_ids = pos_config.env['pos.order.line'].browse([line['id'] for line in results['pos.order.line']])
         order_ids = pos_config.env['pos.order'].browse([order['id'] for order in results['pos.order']])
 
-        self._verify_line_price(line_ids, pos_config)
```

Comment:
this is for the calculation of combo components' prices, not sure if you should remove it. perhaps it should be adapted.